lib/Net/SAML2/Binding/Redirect.pm

```diff
@@ -50,7 +50,12 @@ Arguments:
 The SP's (Service Provider) also known as your application's signing key
 that your application uses to sign the AuthnRequest.  Some IdPs may not
 verify the signature.
-Required with B<param> being C<SAMLRequest>.
+
+Usually required when B<param> is C<SAMLRequest>.
+
+If you don't want to sign the request, you can pass B<< insecure => 1
```

Comment:
I would change `B<<` to `C<<`, but other than that, I'm ok with this PR. Thank you very much!

Reply:
nice catch! thanks